dexcom_reader/database_records.py

```diff
@@ -26,7 +31,7 @@ def _ClassSize(cls):
   @property
   def FMT(self):
     self._CheckFormat()
-    return _ClassFormat()
+    return self._ClassFormat()
```

Comment:
What is the desired functionality here?  It it to call __self._CheckFormat()__ or is it to return the bound method __self._CheckFormat__ without calling it or both?!?